LICENSE.md

```diff
@@ -1,20 +1,57 @@
-### WordPress - Web publishing software
+## Gutenberg
 
     Copyright 2011-2021 by the contributors
```

Comment:
I guess we should also revise the `2011` to the one corresponding to the Gutenberg timeline.

Reply:
Updated the initial year to be 2016 (year of the first commit I could find in this repository) in 5d26b92.